silhouette/test/com/mohiva/play/silhouette/impl/providers/state/CsrfStateItemHandlerSpec.scala

```diff
@@ -0,0 +1,118 @@
+package com.mohiva.play.silhouette.impl.providers.state
+
+import com.mohiva.play.silhouette.api.crypto.CookieSigner
+import com.mohiva.play.silhouette.api.util.IDGenerator
+import com.mohiva.play.silhouette.impl.providers.SocialStateItem
+import com.mohiva.play.silhouette.impl.providers.SocialStateItem.ItemStructure
+import org.specs2.matcher.JsonMatchers
+import org.specs2.mock.Mockito
+import org.specs2.specification.Scope
+import play.api.libs.json.{ Json }
+import play.api.test.{ FakeRequest, PlaySpecification }
+
+import scala.util.Success
+import scala.concurrent.duration._
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.mvc.Cookie
+
+import scala.concurrent.Future
+
+class CsrfStateItemHandlerSpec extends PlaySpecification with Mockito with JsonMatchers {
+
+  "item method of CsrfStateItemHandler" should {
```

Comment:
We should be consistent with the rest of the tests.

We write:
```
"The `item` method" should {
```

The `CsrfStateItemHandler` is not needed here because the file test only one class and therefore it's redundant.

Tests for the same method should be grouped:
```
"The `canHandle` method" should {
  "should return `None` if it can't handle the given `SocialStateItem`" in new Context {
    ...
  }

  "should return true if it can handle the given `ItemStructure`" in new Context {
    ...
  }
}
```
We do not use a point at the end of the sentence.